### PR TITLE
Create zh_CN.lang for ExtraTrees

### PR DESCRIPTION
**Pull request by SihenZhang**
Before merging, I want to let my friend @3TUSK to have a check.

### CODE REVIEW

File: extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang [hunk 1]
```diff
@@ -0,0 +1,686 @@
...
+extratrees.item.food.avacado=鳄梨
+extratrees.item.food.nutmeg=肉豆蔻
+extratrees.item.food.allspice=多香果
+extratrees.item.food.chilli=辣椒
+extratrees.item.food.star_anise=八角
+extratrees.item.food.mango=芒果
+extratrees.item.food.starfruit=杨桃
+extratrees.item.food.candlenut=石栗
+
+extratrees.item.carpentry_hammer.name=假的锤子
```
Comment (3TUSK) [inline extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang:560]: ？

File: extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang [hunk 2]
```diff
@@ -0,0 +1,686 @@
...
+extratrees.item.food.brazil_nut=巴西坚果
+extratrees.item.food.fig=无花果
+extratrees.item.food.acorn=橡果
+extratrees.item.food.elderberry=接骨木果
+extratrees.item.food.olive=橄榄
+extratrees.item.food.gingko_nut=白果
+extratrees.item.food.coffee=咖啡
+extratrees.item.food.osange_orange=桑橙
+extratrees.item.food.clove=丁香
+extratrees.item.food.papayimar=Pahimar牌木瓜
```
Comment (3TUSK) [inline extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang]: Pahimar 牌木瓜

File: extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang [hunk 3]
```diff
@@ -0,0 +1,686 @@
...
+extratrees.fruits.blueberry=蓝莓
+extratrees.fruits.cranberry=蔓越莓
+extratrees.fruits.juniper=杜松子
+extratrees.fruits.gooseberry=鹅莓
+extratrees.fruits.golden_raspberry=金树莓
+
+##################################################
+## Butterfly Species
+
+extrabutterflies.species.whiteadmiral.name=White Admiral
```
Comment (3TUSK) [inline extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang:463]: _Shurg_

File: extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang [hunk 4]
```diff
@@ -0,0 +1,686 @@
...
+##################################################
+## Wood Blocks
+
+< %1$s is name of wood used for door >
+extratrees.block.door.name=%1$s木门
+
+< %1$s is name of wood used for door, %2$s is type of door >
+extratrees.block.door.name.adv=%1$s%2$s木门
+extratrees.block.door.type.standard=
+extratrees.block.door.type.solid=Solid
```
Comment (3TUSK) [inline extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang:275]: Wut

File: extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang [hunk 5]
```diff
@@ -0,0 +1,686 @@
...
+extratrees.block.planks.locust.desc=和毁灭性的虫害毫无关联。|伐木工
+extratrees.block.planks.eucalyptus.desc=一种粉色木材，经常拿来制造木浆。|伐木工
+extratrees.block.planks.purpleheart.desc=此树正是得名于木质上紫色的阴影。|伐木工
+extratrees.block.planks.ash.desc=Gotta catch 'em all!|小智
+extratrees.block.planks.holly.desc=这是最白的木材，记得不要让有尖头的叶子掉进去。|伐木工
+extratrees.block.planks.olive.desc=那引人注目的柔和的绿色也正是其果实的色彩。|伐木工
+extratrees.block.planks.sweetgum.desc=又是一种质地均匀的木材，但它有一些用途。|伐木工
+extratrees.block.planks.rosewood.desc=这又是一种颜色颇深的木材，用于制作国际象棋棋子。|伐木工
+extratrees.block.planks.gingko.desc=和黄杨类似，银杏木颜色很淡。|伐木工
+extratrees.block.planks.pinkivory.desc=我能说些什么呢.它的粉色太浓厚了。|伐木工
```
Comment (3TUSK) [inline extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang]: `.` -> `。`

File: extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang [hunk 6]
```diff
@@ -0,0 +1,686 @@
...
+extratrees.block.planks.forestry.citrus=柑橘树
+extratrees.block.planks.forestry.giganteum=巨杉树
+extratrees.block.planks.forestry.ipe=重蚁树
+extratrees.block.planks.forestry.padauk=紫檀树
+extratrees.block.planks.forestry.cocobolo=黄檀树
+extratrees.block.planks.forestry.zebrawood=斑木树
+
+extratrees.block.planks.ebxl.redwood=红木
+extratrees.block.planks.ebxl.fir=苏格兰冷杉
+extratrees.block.planks.ebxl.acacia=埃克希勒金合欢
```
Comment (3TUSK) [inline extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang:196]: Citation?

**Review by 3TUSK — CHANGES_REQUESTED** [review]
Cannot join BTM Moon server, let's check this...

File: extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang (hunk 6 above)
Comment (3TUSK) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang:196]: Ignore this one.

File: extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang (hunk 4 above)
Comment (SihenZhang) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang:275]: I don't know what are the meanings of these types of door and fence. So I give them up.

Comment (SihenZhang) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang:275]: Have a good idea?

File: extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang (hunk 3 above)
Comment (SihenZhang) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang:463]: Well, I tried to translate the names of butterflies. I failed completely before I finished it.

Comment (SihenZhang) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang:463]: Maybe need an expert in biology especially in lepidopterology.

File: extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang (hunk 1 above)
Comment (SihenZhang) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang:560]: Fake Hammer——假的锤子
I can't find it in the game, so I have no choice but translate it straight.

File: extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang (hunk 3 above)
Comment (3TUSK) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang:463]: Ignore all of them then...

File: extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang (hunk 1 above)
Comment (3TUSK) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang:560]: Ignore this one

File: extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang (hunk 4 above)
Comment (SihenZhang) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang]: Well...
Others are ignored, so what about these?

Comment (SihenZhang) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang]: Well...
Others are ignored, so what about these?

Comment (SihenZhang) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang]: Well...
Others are ignored, so what about these?

Comment (SihenZhang) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang]: Well...
Others are ignored, so what about these?

Comment (SihenZhang) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang]: Well...
Others are ignored, so what about these?

Comment (SihenZhang) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang]: Well...
Others are ignored, so what about these?

Comment (SihenZhang) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang]: Well...
Others are ignored, so what about these?

Comment (SihenZhang) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang]: Well...
Others are ignored, so what about these?

Comment (SihenZhang) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang]: Well...
Others are ignored, so what about these?

Comment (SihenZhang) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang]: Well...
Others are ignored, so what about these?

Comment (SihenZhang) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang]: Well...
Others are ignored, so what about these?

Comment (SihenZhang) [reply extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang]: Forget it, ok?

**Pull request closed without merging**